app/components/course-page/course-stage-step/community-solution-card/content.hbs

```diff
@@ -54,31 +54,32 @@
 {{/each}}
 
 {{#each this.unchangedFileComparisons key="filename" as |unchangedFileComparison|}}
-  <FileContentsCard
-    @headerTooltipText="This file was added in previous stages."
-    @isCollapsible={{true}}
-    @isCollapsed={{not (includes unchangedFileComparison.path this.expandedUnchangedFilePaths)}}
-    @onExpand={{fn this.handleExpandUnchangedFile unchangedFileComparison.path}}
-    @onCollapse={{fn this.handleCollapseUnchangedFile unchangedFileComparison.path}}
-    @filename={{unchangedFileComparison.path}}
-    @code={{unchangedFileComparison.content}}
-    @language={{@solution.language.slug}}
-    data-test-community-solution-unchanged-file
-    {{! scroll-mt-16 accounts for the sticky menu bar}}
-    class="mt-4 text-sm scroll-mt-16"
-  >
-    <:header>
-      <div class="flex items-center justify-between w-full gap-3">
-        <span class="font-mono text-xs text-gray-600 bold">{{unchangedFileComparison.path}}</span>
+  <div class="contents" data-test-community-solution-unchanged-file>
```

Comment:
@VasylMarchuk we tend to avoid adding extra classes / wrappers unless absolutely needed - could the `data-test-community-solution-unchanged-file` just be applied directly to `FileContentsCard`?

Reply:
Unfortunately not in this particular case, I had it originally as you suggest, but then `data-test-file-contents-card-component` ìs not nested into `data-test-community-solution-unchanged-file`, and instead has two of the classes at the same time, and this confuses my `coursePage` pageObject, which expects it to be nested.